pages/reporting/views/index.jsx

```diff
@@ -22,6 +22,15 @@ function DateRange() {
   return `From ${start.format('Do')} to ${end.format('Do MMMM YYYY')}`;
 }
 
+function EndDate() {
+  const { end } = useSelector(state => {
+    return {
+      end: moment(state.model.end)
+    };
+  });
+  return `${end.format('D MMMM YYYY')}`;
```

Comment:
There is a standard formatDate funtion for use in the UIs
```js
const { dateFormat } = require('@asl/pages/constants');
```

```suggestion
  const end = useSelector(({model: {end}}) => end);
  return `${formatDate(end, dateFormat.long)}`;
```

Can you also update the date range formatting above to use the same for consistency